pt_PT/LC_MESSAGES/creatorsRoom.po

```diff
@@ -154,7 +154,7 @@ msgid "VIEW"
 msgstr "Ver"
 
 msgid "FAN_COUNT_FORMAT"
-msgstr "Fans: %s"
+msgstr "Fãs: %s"
```

Comment:
I'm not sure if this character will appear at all on the DS, but we can still merge and correct it in a subsequent PR if it doesn't appear.

Reply:
Oh damn idk wtf I did there. Should be "Fani"

Reply:
The PicToChat keyboard does have that character, so it's safe to say that said character can be added in